src/Microsoft.AspNetCore.Server.Kestrel/Networking/Libuv.cs

```diff
@@ -610,6 +674,9 @@ private static class NativeDarwinMonoMethods
             public static extern void uv_unref(UvHandle handle);
 
             [DllImport("__Internal", CallingConvention = CallingConvention.Cdecl)]
+            public static extern int uv_fileno(UvHandle handle, ref IntPtr socket);
```

Comment:
This is only used on Darwin Mono so may not be needed if the feature is windows specific...